diff --git a/Cargo.toml b/Cargo.toml
index 15b408d..fa5ca92 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -56,3 +56,6 @@ opt-level = 1
 [patch."https://github.com/pop-os/cosmic-protocols"]
 cosmic-protocols = { git = "https://github.com/pop-os/cosmic-protocols//", branch = "main" }
 cosmic-client-toolkit = { git = "https://github.com/pop-os/cosmic-protocols//", branch = "main" }
+
+[profile.release]
+lto = "thin"
